ISISComputingGroup/EPICS-StreamDevice
# Update to 2 8 12

See https://github.com/ISISComputingGroup/IBEX/issues/5283

This updates us to the latest version of stream device (see what's changed at https://github.com/paulscherrerinstitute/StreamDevice/compare/2.8.10...2.8.12).

To test:
* Checkout this branch and run `make` in this directory
* Run the IOC_system tests and confirm they still pass (I just did a few random ones)
